plugin/evm/message/block_sync_summary.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package message
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/codec"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/engine/snowman/block"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/crypto"
+)
+
+var _ Syncable = (*BlockSyncSummary)(nil)
+
+// Codec is the codec manager that contains the codec for BlockSyncSummary and
+// other message types that are used in the network protocol. This is to ensure that the codec
+// version is consistent across all message types and includes the codec for BlockSyncSummary.
+var Codec codec.Manager
+
+func init() {
+	var err error
+	Codec, err = NewCodec(BlockSyncSummary{})
```

Comment:
Similar comment as before, I don't see where we ever use this to marshal/unmarshal interfaces. Why do we register types for this codec?

Reply:
This was a step to reduce diff between subnet-evm. In Subnet-EVM we only use `BlockSyncSummary`. We can argue that they both should have their own codec, but this seemed can be shared.